packages/core/src/mask/MaskSystem.js

```diff
@@ -113,7 +113,7 @@ export default class MaskSystem extends System
         // TODO - may cause issues!
         target.filterArea = maskData.getBounds(true);
 
-        this.renderer.filterSystem.pushFilter(target, alphaMaskFilter);
+        this.renderer.filter.push(target, alphaMaskFilter);
```

Comment:
TypeScript would've caught this! Just saying...

Reply:
IntellIJ idea caught this when i opened the file :)

Reply:
Also, you know I'm all pro-typescript, but our users aren't . I'll make separate TS repository with pixi plugins and typings for v5.

Reply:
Can we just add the typing into this repo directly? It’ll be easier to manage API changes. Kind of what I did with Pixi-filters. Not much extra effort.

Reply:
(remembers nobel laureat) Yes, we can!